views/DataByGender/SelectFeature.html

```diff
@@ -22,5 +22,57 @@
             </div>
         </div>
     </form>
+	{% if data and TypeOfTable == 1%}
+		{% if Gender == 1%}
+			<h2 class="text-center display-5 mt-5">Number of cases, deaths, and recovered for each country for Male</h2>
```

Comment:
Maybe you can just have one header, and inside of the header add the if-else condition below before you close the header.
{% if Gender == 1%}
  male
{% else %}
  female
{% endif %}